opentimelineio/schema/track.py

```diff
@@ -96,6 +96,25 @@ def trimmed_range_of_child_at_index(self, index, reference_space=None):
 
         return self.trim_child_range(child_range)
 
+    def visible_range_of_child(self, child):
+        """The range of this child's media visible to its parent Track.
+        Includes handles revealed by adjacent transitions (if any)."""
+
+        result = child.trimmed_range()
+
+        index = self.index(child)
+        if index > 0:
+            before = self[index-1]
```

Comment:
I'm (hoping) you can use the new `neighbors_of` implementation as a cleaner form of this.

Reply:
Done.